packages/trader/src/AppV2/Components/TradeParameters/trade-parameters.scss

```diff
@@ -13,50 +13,60 @@
         justify-content: space-between;
     }
 
-    &__options__wrapper {
-        display: flex;
-        flex-direction: column;
-        justify-content: center;
-        gap: var(--core-spacing-400);
-
-        &--minimized {
-            align-items: center;
-            flex-direction: row;
-            justify-content: unset;
-            flex-wrap: nowrap;
-            overflow-x: scroll;
-            white-space: nowrap;
-            -ms-overflow-style: none;
-            scrollbar-width: none;
-            max-height: 0px;
-            padding: 0px;
-            transition: transform 0.5s, opacity 0.5s, max-height 0.5s, padding 0.1s;
-
-            // if container has 2 children, then apply style to children
-            &:has(> :last-child:nth-child(2)) {
-                .trade-params__option--minimized {
-                    width: unset;
-                }
-            }
+    &__options {
+        &-info__container {
+            padding: 0 var(--core-spacing-400) var(--core-spacing-400) var(--core-spacing-400);
 
-            &::-webkit-scrollbar {
+            &--isHidden {
```

Comment:
please use conditional rendering instead of unrendering the component by using display:none🙏🏻